src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1032,7 +1032,6 @@ internal void RemoveChild(View child)
 
             Children.Remove(child);
             child.InternalParent = null;
```

Comment:
is this OK to remove minus count?
what if the parent's `LayoutCount` becomes wrong and `LayoutCount` is used in other place.
is there no problem if the `LayoutCount` is not updated and has wrong value when child is removed?

Reply:
Hi， Mr. Ssong, 
Thanks for your mention, I checked the sample again, I found the sample which use the method incorrectly, It's my mistake,  so I'll close the pr.